jsonschema/cli.py

```diff
@@ -69,19 +70,49 @@ def validation_success(self, **kwargs):
 
 @attr.s
 class _PrettyFormatter(object):
+    _MESSAGE_BAR_CHAR = '═'
+    _MESSAGE_CORNER_CHARS = ('╒', '╕')
+    _MESSAGE_FORMAT = '{}══[{}]═══({})'
+    _MESSAGE_MAX_LENGTH = 79
+
+    @classmethod
+    def _json_formatter(cls, x):
+        return json.dumps(x, separators=(',\n', ': '), sort_keys=True)
 
-    _ERROR_MSG = dedent(
-        """\
-        ===[{type}]===({path})===
+    def _message_end_chars(self, header=False):
+        return self._MESSAGE_CORNER_CHARS if header is True else [self._MESSAGE_BAR_CHAR] * 2
```

Comment:
Ideally I'd love to remove the boolean flag for header (and have separate paths for the header and the body lines, i.e. `_header_line` and `_body_line`) -- but as-is if we stick with what's here this should be equivalent to `self._MESSAGE_CORNER_CHARS if header else self._MESSAGE_BAR_CHAR * 2`, no?

Reply:
Ah, I see ok, you're unpacking it below, so you needed the list for Py2, ok. (But yeah the `if True` is unnecessary).

Reply:
I used `is True` to explicitly check for a Boolean value, rather than depend on a truth-y value.  But I can remove it.